Auto update failing for ipfs

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the ipfs package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="bash">
INFO: Updating ipfs [Current version: 0.12.2]
INFO: package being updated to 0.13.0.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

 31 12.8M   31 4176k    0     0  6008k      0  0:00:02 --:--:--  0:00:02 6008k
100 12.8M  100 12.8M    0     0  14.7M      0 --:--:-- --:--:-- --:--:-- 49.0M

You are about to commit these changes:

--------------------
diff --git a/packages/ipfs/build.sh b/packages/ipfs/build.sh
index 45e479acb..24a5642cc 100644
--- a/packages/ipfs/build.sh
+++ b/packages/ipfs/build.sh
@@ -2,10 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://ipfs.io/
 TERMUX_PKG_DESCRIPTION="A peer-to-peer hypermedia distribution protocol"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=0.12.2
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION="0.13.0"
 TERMUX_PKG_SRCURL=https://github.com/ipfs/go-ipfs/releases/download/v${TERMUX_PKG_VERSION}/go-ipfs-source.tar.gz
-TERMUX_PKG_SHA256=eba34d2cc49f7811d087f4259c44c1fecb2b6bd08a6c3fa3b5a26e5fed78c74d
+TERMUX_PKG_SHA256=784207b26b43def178f115451c4cf6f20915bfbbb9d2930113c67ea1a33aa4b3
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_SUGGESTS="termux-services"
 TERMUX_PKG_SERVICE_SCRIPT=("ipfs" "[ ! -d \"${TERMUX_ANDROID_HOME}/.ipfs\" ] && ipfs init --empty-repo 2>&1 && ipfs config --json Swarm.EnableRelayHop false 2>&1 && ipfs config --json Swarm.EnableAutoRelay true 2>&1; exec ipfs daemon --enable-namesys-pubsub 2>&1")
--------------------

ipfs: update to 0.13.0


Not committing to Git!

INFO: Trying to build package.
Running container 'termux-package-builder' from image 'termux/package-builder'...
Creating new container...
Unable to find image 'termux/package-builder:latest' locally
latest: Pulling from termux/package-builder
940122ec7d0c: Pulling fs layer
f8ba241c232b: Pulling fs layer
d82c8aa35ad1: Pulling fs layer
53a6c9889a4e: Pulling fs layer
a3077589c274: Pulling fs layer
d7b5fc08d8b6: Pulling fs layer
53a6c9889a4e: Waiting
a3077589c274: Waiting
d7b5fc08d8b6: Waiting
d82c8aa35ad1: Verifying Checksum
d82c8aa35ad1: Download complete
f8ba241c232b: Download complete
53a6c9889a4e: Verifying Checksum
53a6c9889a4e: Download complete
940122ec7d0c: Verifying Checksum
940122ec7d0c: Download complete
d7b5fc08d8b6: Verifying Checksum
d7b5fc08d8b6: Download complete
940122ec7d0c: Pull complete
f8ba241c232b: Pull complete
d82c8aa35ad1: Pull complete
53a6c9889a4e: Pull complete
a3077589c274: Download complete
a3077589c274: Pull complete
d7b5fc08d8b6: Pull complete
Digest: sha256:bc1a01ec36716b2a34e742fa7b02454836f4bbb60e06f6cf3c1ea2354cff2979
Status: Downloaded newer image for termux/package-builder:latest
5e67ae12935ae6a7ddb4d241ad438f577c21ef1b4ba6bae3afbcf4ec802b5b25
Changed builder uid/gid... (this may take a while)
gpg: key B0076E490B71616B: 10 signatures not checked due to missing keys
gpg: key B0076E490B71616B: public key "Henrik Grimler <henrik@grimler.se>" imported
gpg: Total number processed: 1
gpg:               imported: 1
gpg: no ultimately trusted keys found
gpg (GnuPG) 2.2.20; Copyright (C) 2020 Free Software Foundation, Inc.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

gpg: key 5A897D96E57CF20C: 4 signatures not checked due to missing keys
gpg: key 5A897D96E57CF20C: public key "Termux Releases (Termux automatic builds) <contact@termux.com>" imported
gpg: key 1B388B024A48460E: public key "Termux Releases (Termux automatic builds) <contact@termux.dev>" imported
gpg: Total number processed: 2
gpg:               imported: 2
gpg: marginals needed: 3  completes needed: 1  trust model: pgp
gpg: depth: 0  valid:   1  signed:   2  trust: 0-, 0q, 0n, 0m, 0f, 1u
gpg: depth: 1  valid:   2  signed:   0  trust: 2-, 0q, 0n, 0m, 0f, 0u
gpg: next trustdb check due at 2024-08-25
gpg (GnuPG) 2.2.20; Copyright (C) 2020 Free Software Foundation, Inc.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

termux - building ipfs for arch aarch64...
Downloading https://raw.githubusercontent.com/termux/termux-elf-cleaner/v1.10/termux-elf-cleaner.cpp
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  7569  100  7569    0     0  29682      0 --:--:-- --:--:-- --:--:-- 29682
/home/builder/.termux-build/_cache/termux-elf-cleaner.cpp:24: warning: "DT_AARCH64_BTI_PLT" redefined
   24 | #define DT_AARCH64_BTI_PLT 0x70000001
      | 
In file included from /home/builder/.termux-build/_cache/termux-elf-cleaner.cpp:16:
/usr/include/elf.h:2875: note: this is the location of the previous definition
 2875 | #define DT_AARCH64_BTI_PLT      (DT_LOPROC + 1)
      | 
/home/builder/.termux-build/_cache/termux-elf-cleaner.cpp:25: warning: "DT_AARCH64_PAC_PLT" redefined
   25 | #define DT_AARCH64_PAC_PLT 0x70000003
      | 
In file included from /home/builder/.termux-build/_cache/termux-elf-cleaner.cpp:16:
/usr/include/elf.h:2876: note: this is the location of the previous definition
 2876 | #define DT_AARCH64_PAC_PLT      (DT_LOPROC + 3)
      | 
/home/builder/.termux-build/_cache/termux-elf-cleaner.cpp:26: warning: "DT_AARCH64_VARIANT_PCS" redefined
   26 | #define DT_AARCH64_VARIANT_PCS 0x70000005
      | 
In file included from /home/builder/.termux-build/_cache/termux-elf-cleaner.cpp:16:
/usr/include/elf.h:2877: note: this is the location of the previous definition
 2877 | #define DT_AARCH64_VARIANT_PCS  (DT_LOPROC + 5)
      | 
Downloading https://packages-cf.termux.org/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13133  100 13133    0     0  50706      0 --:--:-- --:--:-- --:--:-- 50706
Downloading https://packages-cf.termux.org/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   3143      0 --:--:-- --:--:-- --:--:--  3143
gpg: Signature made Thu 09 Jun 2022 06:25:59 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: checking the trustdb
gpg: marginals needed: 3  completes needed: 1  trust model: pgp
gpg: depth: 0  valid:   2  signed:   1  trust: 0-, 0q, 0n, 0m, 0f, 2u
gpg: depth: 1  valid:   1  signed:   0  trust: 1-, 0q, 0n, 0m, 0f, 0u
gpg: next trustdb check due at 2024-08-25
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.com>" [ultimate]
Downloading https://packages-cf.termux.org/apt/termux-main/dists/stable/main/binary-aarch64/Packages
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  3 1324k    3 52913    0     0   291k      0  0:00:04 --:--:--  0:00:04  290k
100 1324k  100 1324k    0     0  2499k      0 --:--:-- --:--:-- --:--:-- 2494k
Downloading https://packages-cf.termux.org/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13319  100 13319    0     0  61096      0 --:--:-- --:--:-- --:--:-- 61096
Downloading https://packages-cf.termux.org/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   6310      0 --:--:-- --:--:-- --:--:--  6310
gpg: Signature made Thu 09 Jun 2022 06:56:39 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.com>" [ultimate]
Downloading https://packages-cf.termux.org/apt/termux-root/dists/root/stable/binary-aarch64/Packages
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 45818  100 45818    0     0   121k      0 --:--:-- --:--:-- --:--:--  121k
Downloading https://packages-cf.termux.org/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13119  100 13119    0     0  65268      0 --:--:-- --:--:-- --:--:-- 65268
Downloading https://packages-cf.termux.org/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   3559      0 --:--:-- --:--:-- --:--:--  3559
gpg: Signature made Mon 06 Jun 2022 06:34:21 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.com>" [ultimate]
Downloading https://packages-cf.termux.org/apt/termux-x11/dists/x11/main/binary-aarch64/Packages
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  9  335k    9 32435    0     0  96246      0  0:00:03 --:--:--  0:00:03 95961
100  335k  100  335k    0     0   795k      0 --:--:-- --:--:-- --:--:--  794k
Downloading https://github.com/ipfs/go-ipfs/releases/download/v0.13.0/go-ipfs-source.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 12.8M  100 12.8M    0     0  34.2M      0 --:--:-- --:--:-- --:--:-- 34.2M
Applying ndk-patch: bits-struct_file.h.patch
Applying ndk-patch: dirent.h.patch
Applying ndk-patch: grp.h.patch
Applying ndk-patch: langinfo.h.patch
Applying ndk-patch: linux-fcntl.h.patch
Applying ndk-patch: paths.h.patch
Applying ndk-patch: pwd.h.patch
Applying ndk-patch: redefine-TCSAFLUSH.patch
Applying ndk-patch: semaphore.h.patch
Applying ndk-patch: stdio.h.patch
Applying ndk-patch: stdlib.h.patch
Applying ndk-patch: sys-cdefs.h.patch
Applying ndk-patch: syslog.patch
Applying ndk-patch: unistd.h.patch
Applying ndk-patch: utmp.h.patch
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libamidi.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libbinder_ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libamidi.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libbinder_ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libamidi.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libbinder_ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libicu.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libnativehelper.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libz.so'
Applying patch: go-ipfs-0.4.15_bin_dist_get.patch
Downloading https://golang.org/dl/go1.18.3.linux-amd64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100    80  100    80    0     0    879      0 --:--:-- --:--:-- --:--:--   879

100    75  100    75    0     0    403      0 --:--:-- --:--:-- --:--:--   403

 18  135M   18 25.2M    0     0  53.6M      0  0:00:02 --:--:--  0:00:02 53.6M
100  135M  100  135M    0     0   120M      0  0:00:01  0:00:01 --:--:--  169M
go: downloading github.com/lucas-clemente/quic-go v0.26.0
go: downloading github.com/cheekybits/genny v1.0.0
go: downloading github.com/onsi/ginkgo v1.16.5
go: downloading golang.org/x/net v0.0.0-20220517181318-183a9ca12b87
go: downloading golang.org/x/sys v0.0.0-20220517195934-5e4e11fc645e
go: downloading golang.org/x/crypto v0.0.0-20220518034528-6f7dac969898
go: downloading github.com/go-task/slim-sprig v0.0.0-20210107165309-348f09dbbbc0
go: downloading golang.org/x/tools v0.1.10
go: downloading github.com/marten-seemann/qtls-go1-16 v0.1.5
go: downloading github.com/marten-seemann/qtls-go1-17 v0.1.1
go: downloading github.com/marten-seemann/qtls-go1-18 v0.1.1
go: downloading github.com/nxadm/tail v1.4.8
go: downloading golang.org/x/mod v0.6.0-dev.0.20220106191415-9b9b3d81d5e3
go: downloading gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
go: downloading golang.org/x/xerrors v0.0.0-20220411194840-2f41105eb62f
go: downloading github.com/fsnotify/fsnotify v1.5.1
go: downloading github.com/libp2p/go-libp2p v0.18.1
go: downloading github.com/libp2p/go-libp2p-quic-transport v0.16.1
go: downgraded github.com/libp2p/go-libp2p v0.19.4 => v0.18.1
go: downgraded github.com/libp2p/go-libp2p-quic-transport v0.17.0 => v0.16.1
go: downgraded github.com/lucas-clemente/quic-go v0.27.1 => v0.26.0
go: downloading github.com/cespare/xxhash v1.1.0
go: downloading github.com/ipfs/go-cid v0.2.0
go: downloading github.com/ipfs/go-ipfs-files v0.1.1
go: downloading github.com/ipfs/interface-go-ipfs-core v0.7.0
go: downloading github.com/ipfs/go-ipfs-blockstore v1.2.0
go: downloading github.com/ipfs/go-ipfs-pinner v0.2.1
go: downloading github.com/ipfs/go-ipld-format v0.4.0
go: downloading github.com/ipfs/go-log v1.0.5
go: downloading github.com/jbenet/goprocess v0.1.4
go: downloading github.com/mitchellh/go-homedir v1.1.0
go: downloading github.com/coreos/go-systemd/v22 v22.3.2
go: downloading github.com/hashicorp/go-multierror v1.1.1
go: downloading github.com/ipfs/go-ipfs-cmds v0.8.1
go: downloading github.com/ipfs/go-ipfs-util v0.0.2
go: downloading github.com/ipfs/go-metrics-prometheus v0.0.2
go: downloading github.com/ipfs/go-path v0.3.0
go: downloading github.com/ipfs/go-pinning-service-http-client v0.1.1
go: downloading github.com/ipfs/go-unixfs v0.3.1
go: downloading github.com/libp2p/go-libp2p-core v0.15.1
go: downloading github.com/libp2p/go-libp2p-loggables v0.1.0
go: downloading github.com/libp2p/go-socket-activation v0.1.0
go: downloading github.com/multiformats/go-multiaddr v0.5.0
go: downloading github.com/multiformats/go-multiaddr-dns v0.3.1
go: downloading github.com/prometheus/client_golang v1.12.1
go: downloading go.opentelemetry.io/otel v1.7.0
go: downloading github.com/ipfs/go-merkledag v0.6.0
go: downloading github.com/facebookgo/atomicfile v0.0.0-20151019160806-2de1f203e7d5
go: downloading github.com/ipfs/go-blockservice v0.3.0
go: downloading github.com/ipfs/go-fetcher v1.6.1
go: downloading github.com/ipfs/go-filestore v1.2.0
go: downloading github.com/ipfs/go-graphsync v0.13.1
go: downloading github.com/ipfs/go-ipfs-exchange-interface v0.1.0
go: downloading github.com/ipfs/go-ipfs-provider v0.7.1
go: downloading github.com/ipfs/go-metrics-interface v0.0.1
go: downloading github.com/ipfs/go-mfs v0.2.1
go: downloading github.com/ipfs/go-namesys v0.5.0
go: downloading github.com/libp2p/go-libp2p-kad-dht v0.16.0
go: downloading github.com/libp2p/go-libp2p-pubsub v0.6.1
go: downloading github.com/libp2p/go-libp2p-record v0.1.3
go: downloading github.com/libp2p/go-libp2p-pubsub-router v0.5.0
go: downloading go.uber.org/dig v1.14.0
go: downloading go.uber.org/fx v1.16.0
go: downloading github.com/ipfs/go-datastore v0.5.1
go: downloading github.com/libp2p/go-libp2p-resource-manager v0.3.0
go: downloading github.com/ipfs/go-cidutil v0.1.0
go: downloading github.com/multiformats/go-multibase v0.0.3
go: downloading github.com/cheggaaa/pb v1.0.29
go: downloading github.com/ipfs/go-block-format v0.0.3
go: downloading github.com/ipfs/go-ipld-legacy v0.1.1
go: downloading github.com/ipld/go-car v0.3.2
go: downloading github.com/ipld/go-car/v2 v2.1.1
go: downloading github.com/ipld/go-codec-dagpb v1.4.0
go: downloading github.com/ipld/go-ipld-prime v0.16.0
go: downloading github.com/multiformats/go-multicodec v0.4.1
go: downloading github.com/dustin/go-humanize v1.0.0
go: downloading github.com/ipfs/go-ipfs-exchange-offline v0.2.0
go: downloading github.com/ipfs/go-verifcid v0.0.1
go: downloading golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
go: downloading github.com/elgris/jsondiff v0.0.0-20160530203242-765b5c24c302
go: downloading github.com/ipfs/go-bitswap v0.6.0
go: downloading github.com/ipfs/go-ipfs-keystore v0.0.2
go: downloading github.com/ipfs/tar-utils v0.0.2
go: downloading github.com/libp2p/go-libp2p-kbucket v0.4.7
go: downloading github.com/multiformats/go-multihash v0.1.0
go: downloading github.com/pkg/errors v0.9.1
go: downloading github.com/whyrusleeping/go-sysinfo v0.0.0-20190219211824-4a357d4b90b1
go: downloading github.com/whyrusleeping/multiaddr-filter v0.0.0-20160516205228-e903e4adabd7
go: downloading github.com/ipfs/go-ipns v0.1.2
go: downloading github.com/ipfs/go-ipfs-routing v0.2.1
go: downloading github.com/libp2p/go-libp2p-swarm v0.10.2
go: downloading go.opentelemetry.io/otel/trace v1.7.0
go: downloading github.com/ipfs/go-ipld-cbor v0.0.5
go: downloading github.com/ipfs/go-ipfs-chunker v0.0.5
go: downloading github.com/ipfs/go-ipfs-posinfo v0.0.1
go: downloading github.com/libp2p/go-libp2p-testing v0.9.2
go: downloading contrib.go.opencensus.io/exporter/prometheus v0.4.0
go: downloading github.com/gabriel-vasile/mimetype v1.4.0
go: downloading github.com/libp2p/go-libp2p-http v0.2.1
go: downloading github.com/miekg/dns v1.1.48
go: downloading go.opencensus.io v0.23.0
go: downloading go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.32.0
go: downloading go.uber.org/zap v1.21.0
go: downloading github.com/ipfs/go-unixfsnode v1.4.0
go: downloading github.com/libp2p/go-doh-resolver v0.4.0
go: downloading bazil.org/fuse v0.0.0-20200117225306-7b5117fecadc
go: downloading github.com/ipfs/go-detect-race v0.0.1
go: downloading github.com/blang/semver/v4 v4.0.0
go: downloading github.com/benbjohnson/clock v1.3.0
go: downloading github.com/cenkalti/backoff/v4 v4.1.3
go: downloading github.com/cenkalti/backoff v2.2.1+incompatible
go: downloading github.com/ipfs/go-log/v2 v2.5.1
go: downloading github.com/libp2p/go-libp2p-discovery v0.6.0
go: downloading github.com/libp2p/go-libp2p-mplex v0.7.0
go: downloading github.com/libp2p/go-libp2p-noise v0.4.0
go: downloading github.com/libp2p/go-libp2p-peerstore v0.6.0
go: downloading github.com/libp2p/go-libp2p-routing-helpers v0.2.3
go: downloading github.com/libp2p/go-libp2p-tls v0.4.1
go: downloading github.com/libp2p/go-libp2p-yamux v0.9.1
go: downloading github.com/libp2p/go-tcp-transport v0.5.1
go: downloading github.com/libp2p/go-ws-transport v0.6.0
go: downloading github.com/wI2L/jsondiff v0.2.0
go: downloading github.com/stretchr/testify v1.7.1
go: downloading github.com/jbenet/go-temp-err-catcher v0.1.0
go: downloading github.com/opentracing/opentracing-go v1.2.0
go: downloading github.com/ipfs/go-ds-badger v0.3.0
go: downloading github.com/ceramicnetwork/go-dag-jose v0.1.0
go: downloading github.com/ipfs/go-ds-flatfs v0.5.1
go: downloading github.com/ipfs/go-ipld-git v0.1.1
go: downloading github.com/ipfs/go-ds-leveldb v0.5.0
go: downloading github.com/syndtr/goleveldb v1.0.0
go: downloading github.com/ipfs/go-ds-measure v0.2.0
go: downloading github.com/ipfs/go-fs-lock v0.0.7
go: downloading github.com/jbenet/go-random v0.0.0-20190219211222-123a90aedc0c
go: downloading go.opentelemetry.io/otel/exporters/jaeger v1.7.0
go: downloading go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc v1.7.0
go: downloading go.opentelemetry.io/otel/exporters/otlp/otlptrace v1.7.0
go: downloading go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp v1.7.0
go: downloading go.opentelemetry.io/otel/exporters/stdout/stdouttrace v1.7.0
go: downloading go.opentelemetry.io/otel/exporters/zipkin v1.7.0
go: downloading go.opentelemetry.io/otel/sdk v1.7.0
go: downloading github.com/spaolacci/murmur3 v1.1.0
go: downloading github.com/multiformats/go-varint v0.0.6
go: downloading github.com/crackcomm/go-gitignore v0.0.0-20170627025303-887ab5e44cc3
go: downloading github.com/hashicorp/golang-lru v0.5.4
go: downloading github.com/ipfs/bbloom v0.0.4
go: downloading github.com/ipfs/go-ipfs-ds-help v1.1.0
go: downloading github.com/OneOfOne/xxhash v1.2.2
go: downloading go.uber.org/atomic v1.9.0
go: downloading github.com/hashicorp/errwrap v1.1.0
go: downloading github.com/Kubuxu/go-os-helper v0.0.1
go: downloading github.com/texttheater/golang-levenshtein v0.0.0-20180516184445-d188e65d659e
go: downloading github.com/rs/cors v1.7.0
go: downloading github.com/mr-tron/base58 v1.2.0
go: downloading github.com/gogo/protobuf v1.3.2
go: downloading github.com/google/uuid v1.3.0
go: downloading github.com/beorn7/perks v1.0.1
go: downloading github.com/cespare/xxhash/v2 v2.1.2
go: downloading github.com/golang/protobuf v1.5.2
go: downloading github.com/prometheus/client_model v0.2.0
go: downloading github.com/prometheus/common v0.33.0
go: downloading github.com/prometheus/procfs v0.7.3
go: downloading google.golang.org/protobuf v1.28.0
go: downloading github.com/go-logr/logr v1.2.3
go: downloading github.com/go-logr/stdr v1.2.2
go: downloading github.com/ipfs/go-ipfs-blocksutil v0.0.1
go: downloading github.com/whyrusleeping/base32 v0.0.0-20170828182744-c30ac30633cc
go: downloading github.com/libp2p/go-flow-metrics v0.0.3
go: downloading github.com/btcsuite/btcd v0.22.1
go: downloading github.com/libp2p/go-openssl v0.0.7
go: downloading github.com/minio/sha256-simd v1.0.0
go: downloading github.com/libp2p/go-msgio v0.2.0
go: downloading github.com/whyrusleeping/timecache v0.0.0-20160911033111-cfcb2f1abfee
go: downloading github.com/libp2p/go-libp2p-blankhost v0.3.0
go: downloading github.com/libp2p/go-libp2p-connmgr v0.2.4
go: downloading github.com/libp2p/zeroconf/v2 v2.1.1
go: downloading github.com/libp2p/go-eventbus v0.2.1
go: downloading github.com/libp2p/go-libp2p-nat v0.1.0
go: downloading github.com/libp2p/go-netroute v0.2.0
go: downloading github.com/multiformats/go-multistream v0.3.0
go: downloading go.uber.org/multierr v1.8.0
go: downloading go.uber.org/goleak v1.1.12
go: downloading gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c
go: downloading github.com/jbenet/go-cienv v0.1.0
go: downloading github.com/pbnjay/memory v0.0.0-20210728143218-7b4eea64cf58
go: downloading github.com/multiformats/go-base32 v0.0.4
go: downloading github.com/multiformats/go-base36 v0.1.0
go: downloading github.com/mattn/go-runewidth v0.0.4
go: downloading github.com/fatih/color v1.9.0
go: downloading github.com/mattn/go-colorable v0.1.4
go: downloading golang.org/x/exp v0.0.0-20210615023648-acb5c1269671
go: downloading github.com/ipld/go-ipld-prime/storage/bsadapter v0.0.0-20211210234204-ce2a1c70cd73
go: downloading github.com/frankban/quicktest v1.14.2
go: downloading github.com/polydawn/refmt v0.0.0-20201211092308-30ac6d18308e
go: downloading github.com/google/go-cmp v0.5.7
go: downloading github.com/mgutz/ansi v0.0.0-20170206155736-9520e82c474b
go: downloading github.com/ipfs/go-ipfs-delay v0.0.1
go: downloading github.com/libp2p/go-libp2p-netutil v0.1.0
go: downloading github.com/google/gopacket v1.1.19
go: downloading github.com/libp2p/go-libp2p-xor v0.1.0
go: downloading github.com/libp2p/go-buffer-pool v0.0.2
go: downloading github.com/alecthomas/units v0.0.0-20210927113745-59d0afb8317a
go: downloading github.com/golang/mock v1.6.0
go: downloading github.com/libp2p/go-conn-security-multistream v0.3.0
go: downloading github.com/libp2p/go-libp2p-transport-upgrader v0.7.1
go: downloading github.com/libp2p/go-stream-muxer-multistream v0.4.0
go: downloading github.com/multiformats/go-multiaddr-fmt v0.1.0
go: downloading github.com/whyrusleeping/cbor-gen v0.0.0-20210219115102-f37d292932f2
go: downloading github.com/whyrusleeping/chunker v0.0.0-20181014151217-fe64bd25879f
go: downloading google.golang.org/grpc v1.46.0
go: downloading github.com/prometheus/statsd_exporter v0.21.0
go: downloading github.com/libp2p/go-libp2p-gostream v0.3.0
go: downloading github.com/felixge/httpsnoop v1.0.3
go: downloading go.opentelemetry.io/otel/metric v0.30.0
go: downloading github.com/ipfs/go-peertaskqueue v0.7.1
go: downloading github.com/mattn/go-isatty v0.0.14
go: downloading github.com/libp2p/go-mplex v0.7.0
go: downloading github.com/flynn/noise v1.0.0
go: downloading github.com/klauspost/compress v1.15.1
go: downloading github.com/libp2p/go-yamux/v3 v3.1.2
go: downloading github.com/whyrusleeping/mdns v0.0.0-20190826153040-b9b60ed33aa9
go: downloading github.com/raulk/go-watchdog v1.2.0
go: downloading github.com/libp2p/go-libp2p-asn-util v0.2.0
go: downloading github.com/libp2p/go-libp2p-circuit v0.6.0
go: downloading github.com/libp2p/go-reuseport v0.1.0
go: downloading github.com/libp2p/go-reuseport-transport v0.1.0
go: downloading github.com/marten-seemann/tcp v0.0.0-20210406111302-dfbc87cc63fd
go: downloading github.com/mikioh/tcpinfo v0.0.0-20190314235526-30a79bb1804b
go: downloading github.com/gorilla/websocket v1.5.0
go: downloading github.com/tidwall/gjson v1.14.0
go: downloading github.com/dgraph-io/badger v1.6.2
go: downloading gopkg.in/square/go-jose.v2 v2.5.1
go: downloading github.com/warpfork/go-testmark v0.9.0
go: downloading pgregory.net/rapid v0.4.7
go: downloading github.com/alexbrainman/goissue34681 v0.0.0-20191006012335-3fc7a47baff5
go: downloading github.com/davecgh/go-spew v1.1.1
go: downloading github.com/pmezard/go-difflib v1.0.0
go: downloading gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
go: downloading go4.org v0.0.0-20200411211856-f5505b9728dd
go: downloading go.opentelemetry.io/otel/exporters/otlp/internal/retry v1.7.0
go: downloading go.opentelemetry.io/proto/otlp v0.16.0
go: downloading google.golang.org/genproto v0.0.0-20211118181313-81c1377c94b1
go: downloading github.com/openzipkin/zipkin-go v0.4.0
go: downloading golang.org/x/term v0.0.0-20210927222741-03fcf44c2211
go: downloading golang.org/x/oauth2 v0.0.0-20220223155221-ee480838109b
go: downloading github.com/matttproud/golang_protobuf_extensions v1.0.1
go: downloading github.com/spacemonkeygo/spacelog v0.0.0-20180420211403-2296661a0572
go: downloading github.com/klauspost/cpuid/v2 v2.0.12
go: downloading github.com/libp2p/go-cidranger v1.1.0
go: downloading github.com/libp2p/go-nat v0.1.0
go: downloading github.com/coreos/go-semver v0.3.0
go: downloading github.com/btcsuite/btcd/chaincfg/chainhash v1.0.1
go: downloading github.com/kr/pretty v0.3.0
go: downloading github.com/petar/GoLLRB v0.0.0-20210522233825-ae3b015fd3e9
go: downloading github.com/whyrusleeping/cbor v0.0.0-20171005072247-63513f603b11
go: downloading github.com/warpfork/go-wish v0.0.0-20200122115046-b9ea61034e4a
go: downloading gopkg.in/yaml.v2 v2.4.0
go: downloading github.com/cskr/pubsub v1.0.2
go: downloading github.com/whyrusleeping/go-keyspace v0.0.0-20160322163242-5b898ac5add1
go: downloading github.com/ipfs/go-bitfield v1.0.0
go: downloading github.com/libp2p/go-libp2p-pnet v0.2.0
go: downloading github.com/smartystreets/goconvey v1.6.4
go: downloading github.com/go-kit/log v0.2.0
go: downloading github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da
go: downloading github.com/ipfs/go-ipfs-pq v0.0.2
go: downloading github.com/hannahhoward/go-pubsub v0.0.0-20200423002714-8d62886cc36e
go: downloading github.com/Stebalien/go-bitfield v0.0.1
go: downloading github.com/tv42/httpunix v0.0.0-20191220191345-2ba4b9c3382c
go: downloading github.com/onsi/gomega v1.16.0
go: downloading github.com/francoispqt/gojay v1.2.13
go: downloading github.com/containerd/cgroups v1.0.3
go: downloading github.com/elastic/gosigar v0.14.2
go: downloading github.com/raulk/clock v1.1.0
go: downloading github.com/opencontainers/runtime-spec v1.0.2
go: downloading github.com/mikioh/tcpopt v0.0.0-20190314235656-172688c1accc
go: downloading github.com/mikioh/tcp v0.0.0-20190314235350-803a9b46060c
go: downloading github.com/tidwall/match v1.1.1
go: downloading github.com/tidwall/pretty v1.2.0
go: downloading github.com/dgraph-io/ristretto v0.0.2
go: downloading github.com/stretchr/objx v0.1.1
go: downloading github.com/grpc-ecosystem/grpc-gateway/v2 v2.7.0
go: downloading github.com/huin/goupnp v1.0.3
go: downloading github.com/jackpal/go-nat-pmp v1.0.2
go: downloading github.com/koron/go-ssdp v0.0.2
go: downloading github.com/kr/text v0.2.0
go: downloading github.com/rogpeppe/go-internal v1.6.1
go: downloading github.com/minio/blake2b-simd v0.0.0-20160723061019-3f5f724cb5b1
go: downloading lukechampine.com/blake3 v1.1.7
go: downloading github.com/davidlazar/go-crypto v0.0.0-20200604182044-b73af7476f6c
go: downloading github.com/jtolds/gls v4.20.0+incompatible
go: downloading github.com/smartystreets/assertions v1.0.0
go: downloading github.com/go-logfmt/logfmt v0.5.1
go: downloading github.com/docker/go-units v0.4.0
go: downloading github.com/godbus/dbus/v5 v5.1.0
go: downloading github.com/AndreasBriese/bbloom v0.0.0-20190825152654-46b345b51c96
go: downloading github.com/dgryski/go-farm v0.0.0-20190423205320-6a90982ecee2
go: downloading github.com/golang/snappy v0.0.4
go: downloading google.golang.org/appengine v1.6.6
go: downloading github.com/gopherjs/gopherjs v0.0.0-20190430165422-3e4dfb77656c
go: downloading github.com/golang/glog v1.0.0
go: downloading golang.org/x/text v0.3.7
go version go1.18.3 linux/amd64
bin/check_go_version 1.17
plugin/loader/preload.sh > plugin/loader/preload.go
go fmt plugin/loader/preload.go >/dev/null
go build "-asmflags=all='-trimpath=/home/builder/.termux-build/ipfs/build'" "-gcflags=all='-trimpath=/home/builder/.termux-build/ipfs/build'" -ldflags="-X "github.com/ipfs/go-ipfs".CurrentCommit=c9d51bbe0" -o "cmd/ipfs/ipfs" "github.com/ipfs/go-ipfs/cmd/ipfs"
# github.com/ipfs/go-ipfs/core/node/libp2p
core/node/libp2p/relay.go:59:42: cannot use autorelay.WithStaticRelays(static) (value of type autorelay.StaticRelayOption) as type autorelay.Option in argument to append
core/node/libp2p/relay.go:60:52: undefined: autorelay.WithCircuitV1Support
core/node/libp2p/relay.go:63:52: undefined: autorelay.WithPeerSource
core/node/libp2p/relay.go:65:56: cannot use autoRelayOpts (variable of type []autorelay.Option) as type []autorelay.StaticRelayOption in argument to libp2p.EnableAutoRelay
make: *** [cmd/ipfs/Rules.mk:22: cmd/ipfs/ipfs] Error 2
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2022-06-10 00:30:37 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/2471934779">2471934779</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/ipfs/build.sh
+++ b/packages/ipfs/build.sh
@@ -2,10 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://ipfs.io/
 TERMUX_PKG_DESCRIPTION="A peer-to-peer hypermedia distribution protocol"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=0.12.2
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION=0.13.0
 TERMUX_PKG_SRCURL=https://github.com/ipfs/go-ipfs/releases/download/v${TERMUX_PKG_VERSION}/go-ipfs-source.tar.gz
-TERMUX_PKG_SHA256=eba34d2cc49f7811d087f4259c44c1fecb2b6bd08a6c3fa3b5a26e5fed78c74d
+TERMUX_PKG_SHA256=784207b26b43def178f115451c4cf6f20915bfbbb9d2930113c67ea1a33aa4b3
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_SUGGESTS="termux-services"
 TERMUX_PKG_SERVICE_SCRIPT=("ipfs" "[ ! -d \"${TERMUX_ANDROID_HOME}/.ipfs\" ] && ipfs init --empty-repo 2>&1 && ipfs config --json Swarm.EnableRelayHop false 2>&1 && ipfs config --json Swarm.EnableAutoRelay true 2>&1; exec ipfs daemon --enable-namesys-pubsub 2>&1")
@@ -19,16 +18,10 @@ termux_step_make() {
 	cp -a "${TERMUX_PKG_SRCDIR}" "${GOPATH}/src/github.com/ipfs/go-ipfs"
 	cd "${GOPATH}/src/github.com/ipfs/go-ipfs"
 
-	# Needed to build against go 1.18.
-	# TODO: Remove after https://github.com/ipfs/go-ipfs/issues/8819 is resolved.
-	go get github.com/lucas-clemente/quic-go@v0.26.0
-	go mod edit -go=1.18 # qo-qtls needs go 1.18
-	go mod tidy && go mod vendor
-
 	make build
 
 	# Fix folders without write permissions preventing which fails repeating builds:
-	cd $TERMUX_PKG_BUILDDIR
+	cd "$TERMUX_PKG_BUILDDIR"
 	find . -type d -exec chmod u+w {} \;
 }
 

